apps/files_sharing/appinfo/routes.php

```diff
@@ -82,6 +82,16 @@
 			'url' => '/settings/defaultAccept',
 			'verb' => 'PUT',
 		],
+		[
+			'name' => 'Settings#setShareFolder',
+			'url' => '/settings/shareFolder',
+			'verb' => 'PUT',
+		],
+		[
+			'name' => 'Settings#resetShareFolder',
+			'url' => '/settings/resetShareFolder',
+			'verb' => 'PUT',
```

Comment:
```suggestion
			'name' => 'Settings#resetShareFolder',
			'url' => '/settings/shareFolder',
			'verb' => 'DELETE',
```

Would fit the API more properly I think :)